tools/package_tor.js

```diff
@@ -37,11 +45,11 @@ if (!fs.existsSync(torPath)) {
 
 var sha512Tor
 if (isDarwin) {
-  sha512Tor = 'af09b7f31fcb3996db3883dff0c20187fdb9653fb5d207dc7e8d3d8cd9de90f2908e47f4414c03b5633a94183425e0c9804316ece353f8174ec4897559932a4e'
+  sha512Tor = '49c7bf2bb648faca53c578b69a8e85e6e817ddd5d9991cc3c61d6d445dcca8f4a745053a57f4093bb161168d1f5d6dba005a9d969a75c286013a37c273b3bb64'
 } else if (isLinux) {
-  sha512Tor = '8e78aa316aef74b6c68983cc3a82ec77e0805001faa1bc0bcf6458227af56929cdf9bd6d48626026aba5a48c82a71eaab15588a6c744b02c2e0be990e73e034b'
+  sha512Tor = 'a93c447ede02df5ea5992c1d1c18ea16544b82513a0c8f21cd818c7724b2c489970c749444e5098f1c57f6e4db44106825ad23a41dbb277e2576d22ae38e6ea6'
 } else {
-  sha512Tor = '0fbd88d590069fdc243fcdcc84b8aa10caa921fc11d7e776334f0cbd5b0095b21046adfd291be61dc414c232bc1ff22e7e8142b0af1d20e71154f8de66be83ab'
+  sha512Tor = 'bd25dcd1e6ca8c1048eaac584aa462a030a302d9f13e55d7f13f4003ec24b9e6ac1396fa62b2bda2dfbd1babed69fe205c605bf978c0779995f4817add84f981'
```

Comment:
why does the windows hash change if the version is the same?

Reply:
I created a new bundle for all, verified that the hashes for the content matches. May be related to `timestamp` or `ordering of files`